### DISCUSSION TITLE
Transferring mania skin from osu!stable to lazer causes the judgement bursts to change

### DISCUSSION THREAD

**Discussion by shnomp** [ppy/osu#18001] [Q&A]
My mania skin in osu!stable uses custom judgment bursts that don't show at all when I play 4k on lazer, although they do show for 7k and I'm not sure why. Instead, the judgment bursts from the default skin are used. The judgment bursts are also much lower on the screen compared to where they were in osu!stable. Is there any way that I can fix this issue manually, or is it some sort of bug?

**Comment by bdach** [comment]
Please link the skin.

**Reply by shnomp** [reply]
I actually managed to fix the judgment bursts to the ones that I use normally, although the judgments are still much lower on the screen.
This is the skin in google drive: https://drive.google.com/drive/folders/1fbdz95k0g6BqXXE8lkJuHMzysZzXIvOX?usp=sharing

**Reply by shnomp** [reply]
Strangely enough, I managed to fix the judgment position as well by changing the positioning in the skin.ini file, but I had to make the value much lower than it is for my skin on osu!stable in order to get the judgments in a similar position, which I find sort of odd.

**Reply by frenzibyte** [reply]
Would be helpful if you can attach screenshots of the issue you have in lazer and the expected positioning in stable, along with referencing which settings are you referring to. I'm slightly confused here.

I'm also not sure how you fixed the judgement bursts as you mentioned, if it's broken in lazer as-is but works on stable then something probably needs resolution. Can you state what you've changed to "fix" it in case that comes in handy.
